src/App.tsx

```diff
@@ -626,13 +670,15 @@ export default class App extends React.Component<
 
         <div className="lf-component-container lf-right-button">
           <lf-login redirect_uri={this.REDIRECT_URI}
+            scope={this.SCOPE}
             authorize_url_host_name={this.HOST_NAME} redirect_behavior="Replace" client_id={this.CLIENT_ID}
             ref={this.loginComponent}>
           </lf-login>
         </div>
 
         <div hidden={!this.state?.isLoggedIn}>
-          {this.state?.showNewFolderDialog && <Modal onClose={this.hideDialog.bind(this)} errorMessage={this.state?.popupErrorMessage} />}
+          {this.state?.showNewFolderDialog && <NewFolderModal onClose={this.hideNewFolderDialog.bind(this)} errorMessage={this.state?.popupErrorMessage} />}
+          {this.state?.showEditColumnsDialog && <EditColumnsModal onClose={this.hideEditColumnsDialog.bind(this)} errorMessage={this.state?.popupErrorMessage} initialColumnsSelected={this.state?.selectedColumns} allColumnOptions={this.state?.allPossibleColumns} />}
```

Comment:
double check the use case for error Message for the column selector. for new foler modal, since we make an api call to create a new folder, the error message is shown if the API call doesn't return 200. Column selector doesn't have need to call an API, and the error handling should be relatively simple.